src/LocalField/neq.jl

```diff
@@ -0,0 +1,216 @@
+#export: degree_relative, random_elem, one_root, norm_equation  
+
+
+degree_relative(L::Hecke.LocalField,K::Hecke.LocalField) = Int(absolute_degree(L)//absolute_degree(K)
+)
+
+ degree_relative(L::Hecke.RelFinField,K:: Hecke.RelFinField)  = Int(absolute_degree(L)//absolute_degree(K))
+degree_relative(L::Hecke.RelFinField,K::FqNmodFiniteField)  = Int(absolute_degree(L)//absolute_degree(K))
+
+degree_relative(L::FqNmodFiniteField, K::Hecke.Nemo.GaloisField)  = Int(absolute_degree(L)//absolute_degree(K))
+
+
+##############################################
+#random element with small coefficients
+##############################################
+
+function random_elem(L::Hecke.LocalField)
+     b = basis(L)
+     n = degree(L)
+     r = [rand(1:5*n) for i in 1:n]
+return sum( [ r[i]*b[i] for i in 1:n])
+end
+
+
+function random_elem(L::FlintQadicField)
+     b = basis_qadic(L)
+     n = degree(L)
+     r = [rand(1:5*n) for i in 1:n]
+return sum( [ r[i]*b[i] for i in 1:n])
+end
+
+
+
+
+
+function one_root( f::fq_nmod_poly, F:: Hecke.RelFinField)
+   g = polynomial(F, [coeff(f,i) for i = 0:degree(f) ] )
+   fac = factor(g)
+   if length(fac)  == 1
+      return []
+   end
+   r = [x for x in fac ][1][1]
```

Comment:
Huh? Perhaps like this?
```suggestion
   r = first(fac)[1]
```